packages/design-picker/src/components/theme-preview-container/style.scss

```diff
@@ -0,0 +1,104 @@
+@import '@automattic/onboarding/styles/mixins';
+
+.theme-preview-container {
+	display: flex;
+	flex-direction: row;
+}
+
+.theme-preview-container__sidebar {
+	width: 343px;
+
+	> div {
+		margin: 2rem 0 0;
+
+		h2 {
+			color: var( --studio-gray-100 );
+			font-size: $font-body-small;
+			font-weight: 500;
+			margin-bottom: 0.5rem;
+		}
+	}
+
+	.theme-preview-container__sidebar-title {
+		margin: 0 0 16px;
+
+		h1 {
+			@include onboarding-font-recoleta;
+			color: var( --studio-gray-100 );
+			font-size: 2rem;
+			line-height: 2rem;
+		}
+	}
+
+	.theme-preview-container__sidebar-description {
+		p {
+			color: var( --studio-gray-80 );
+			font-size: 0.875rem;
+			line-height: 1.25rem;
+		}
+	}
+
+	.theme-preview-container__sidebar-variations {
+		.theme-preview-container__sidebar-variations-grid {
+			display: grid;
+			gap: 8px;
+			grid-template-columns: repeat( 2, 1fr );
+			margin: 0 -5px;
+		}
+	}
+
+	.theme-preview-container__sidebar-support {
+		p {
+			color: var( --studio-gray-80 );
+			font-size: $font-body-small;
+		}
+	}
+
+	.theme-preview-container__sidebar-features {
+		.theme-preview-container__sidebar-features-list {
+			align-items: flex-start;
+			border-radius: 4px;
+			display: flex;
+			flex-wrap: wrap;
+			gap: 12px 4px;
+
+			> div {
+				background-color: var( --studio-gray-5 );
+				border-radius: 4px;
+				color: var( --studio-gray-80 );
+				font-size: 0.75rem;
+				font-weight: 500;
+				line-height: 1.25rem;
+				padding: 0 10px;
+				text-align: center;
+			}
+		}
+	}
+}
+
+.theme-preview-container__sticky-variations {
+	align-items: center;
+	backdrop-filter: blur( 15px );
+	background: rgba( 255, 255, 255, 0.9 );
+	border-bottom: 1px solid rgba( 0, 0, 0, 0.05 );
+	box-shadow: -4px 0 8px rgba( 0, 0, 0, 0.07 );
+	box-sizing: border-box;
+	display: flex;
+	height: 96px;
+	inset-inline-start: 0;
+	inset-inline-end: 0;
+	overflow: auto;
+	padding: 8px 24px 0;
+	position: fixed;
+	top: 0;
+	z-index: 30;
```

Comment:
Could we add the z-index to [here](https://github.com/Automattic/wp-calypso/blob/8d0fe3f90dc85a490ea7334f8df73d0d8f1e694b/client/assets/stylesheets/shared/functions/_z-index.scss#L180) to make it well-organized as much as possible. Then we could use the following way to get it

```scss
z-index: z-index( '.is-section-stepper', '.theme-preview-container__sticky-variations' );
```